.gitignore

```diff
@@ -3,3 +3,6 @@ Cargo.lock
 /target
 **/*.rs.bk
 bundle.ll
+/.vscode
+integration/subxt-tests/contracts/
+integration/subxt-tests/target/
```

Comment:
It might be easier to keep this in `integraton/subxt-tests/.gitignore`

Reply:
Agree! I'll kept it in it's respective folder.